packages/cwd-macros/src/lib.rs

```diff
@@ -138,8 +174,15 @@ pub fn token_query(metadata: TokenStream, input: TokenStream) -> TokenStream {
     let mut ast: DeriveInput = parse_macro_input!(input);
     match &mut ast.data {
         syn::Data::Enum(DataEnum { variants, .. }) => {
-            let info: Variant = syn::parse2(quote! { TokenContract {} }).unwrap();
+            let mut info: Variant = syn::parse2(quote! { TokenContract {} }).unwrap();
+
+            let info_attr = syn::Attribute::parse_outer
+                .parse2(quote! {
+                    #[returns(Addr)]
```

Comment:
to avoid lint issues i think we need to make all of these `#[returns(...)]` decorators use the absolute path. for this one:

```suggestion
                    #[returns(::cosmwasm_std::Addr)]
```

Reply:
ah, sweet emergent complexity. this does not actually work for queries here that return stuff from `cwd-interface`.

for example, `#[returns(::cwd_interface::voting::VotingPowerAtHeightResponse)` breaks when we derive that macro inside the `cwd_interface` crate as there `cwd_interface` has no meaning. for that particular crate, we instead want to do `#[returns(::crate::voting::VotingPowerAtHeightResponse)`.